### PR TITLE
chore: TTL and S3 config related changes

### PR DESCRIPTION
**Pull request by prashant-shahi**
This PR includes TTL and s3 config related changes for the new release.

Merge this PR after merging #1173

### CODE REVIEW

File: frontend/src/container/GeneralSettings/index.tsx [hunk 1]
```diff
@@ -5,37 +5,66 @@ import Spinner from 'components/Spinner';
...
-	const [getRetentionPeriodApiResponse, getDisksResponse] = useQueries([
+	const [
+		getRetentionPeriodMetricsApiResponse,
+		getRetentionPeriodTracesApiResponse,
+		getDisksResponse,
+	] = useQueries([
 		{
-			queryFn: getRetentionPeriodApi,
-			queryKey: 'getRetentionPeriodApi',
+			queryFn: (): Promise<
```
Comment (palash-signoz) [inline frontend/src/container/GeneralSettings/index.tsx:21]: Can we make a common type here as we are changing metrics and traces only

File: frontend/src/container/GeneralSettings/index.tsx [hunk 2]
```diff
@@ -5,37 +5,66 @@ import Spinner from 'components/Spinner';
...
 			<Typography>
-				{getRetentionPeriodApiResponse.data?.error ||
+				{getRetentionPeriodTracesApiResponse.data?.error ||
 					getDisksResponse.data?.error ||
 					t('something_went_wrong')}
 			</Typography>
 		);
 	}
 
 	if (
```
Comment (palash-signoz) [inline frontend/src/container/GeneralSettings/index.tsx:59]: would be great to write some comments

**Review by palash-signoz — CHANGES_REQUESTED** [review]

File: frontend/public/locales/en-GB/translation.json [hunk 3]
```diff
@@ -17,12 +17,22 @@
 	"settings": {
 		"total_retention_period": "Total Retention Period",
 		"move_to_s3": "Move to S3\n(should be lower than total retention period)",
-		"retention_success_message": "Congrats. The retention periods for {{name}} has been updated successfully.",
+		"status_message": {
```
[the remark this answers is not in the document]
Comment (prashant-shahi) [reply frontend/public/locales/en-GB/translation.json:20]: @pranshuchittora ^^

File: frontend/src/container/GeneralSettings/index.tsx (hunk 1 above)
Comment (prashant-shahi) [reply frontend/src/container/GeneralSettings/index.tsx:21]: @pranshuchittora ^^

File: frontend/src/container/GeneralSettings/index.tsx (hunk 2 above)
Comment (prashant-shahi) [reply frontend/src/container/GeneralSettings/index.tsx:59]: @pranshuchittora ^^

Comment (pranshuchittora) [reply frontend/src/container/GeneralSettings/index.tsx:59]: :ack: _(with Axolo)_

**Pull request merged**